plugins/iis_logs.yaml

```diff
@@ -54,6 +54,13 @@ parameters:
     description: When enabled will preserve the original log message on the body in a `raw_log` key
```

Comment:
Do we need to update the description here? Seems like where it retains the original message is dependent on `parse_to`, and no longer just `body`

Reply:
Good call I missed this.